### ISSUE TITLE
markdown improperly parses images with links in their alt text

### ISSUE THREAD

**Issue by BrennanColberg** [BrennanColberg/brennancolbergdotcom#7]
if markdown looks like this...

```markdown
![alt with a [link](https://brennancolberg.com)]($IMAGE_URL)
```

...it should render the image, and that link should be in the alt text (displayed in hover and below the image itself).

What actually happens is that the first `]` in the link "closes" the image tag, failing to render the image entirely and instead plopping the remaining alt text into the DOM directly.

(Might this require switching to a different Markdown renderer, or making my own?)

**Comment by BrennanColberg** [comment]
interesting, syntax parsing above makes it seem like this is proper for Markdown; at least, GitHub does it "incorrectly" too. Perhaps I should be setting a simpler alt text and making the image label a different thing right under it in markdown
